admin/find-gc.el

```diff
@@ -56,7 +56,7 @@ Each entry has the form (FUNCTION . FUNCTIONS-IT-CALLS).")
     "term.c" "cm.c" "emacs.c" "keyboard.c" "macros.c"
     "keymap.c" "sysdep.c" "buffer.c" "filelock.c"
     "insdel.c" "marker.c" "minibuf.c" "fileio.c"
-    "dired.c" "cmds.c" "casefiddle.c"
+    "dired.c" "casefiddle.c"
```

Comment:
Woah, I had no idea this file existed. We should look at updating it to look in Rust. I've created #747.